infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/PodTerminationGracePeriodProvisioner.java

```diff
@@ -31,6 +34,7 @@
  */
 public class PodTerminationGracePeriodProvisioner implements ConfigurationProvisioner {
   private final long graceTerminationPeriodSec;
+  private final long graceTerminationPeriodAsyncPvc = 60;
```

Comment:
would be nice to add the relevant docs with clarification why we can not fall back on the default `graceTerminationPeriodSec` in case of async storage.

Reply:
Should we make this property configurable?

Reply:
I don't think we need to configure this, at least now

Reply:
agree, for experimental feature this should be fine

Reply:
Added some comments, hope this help. Need to find better solution for such behavior in the future.